docs/gettingstarted/query_test_report.rst

```diff
@@ -268,6 +272,14 @@ where ``--count`` is number of rows that that get printed. Shown below is the ou
 The ``--count`` option also works with terse mode ``--terse``, shown below is the same output 
 
 .. command-output:: buildtest report --terse --count 2
+Color Output
+-------------
+
+If you would like to change the output color for either ``buildtest report`` and ``buildtest report summary`` you can use the ``--color`` option.
+This will print the output of the previous two commands in the provided color. The color must be one of the supported colors by rich console.
+
+.. command-output:: buildtest report --color red 
+.. command-output:: buildtest report --color red summary
```

Comment:
Don't we want this section under the section you had previously defined `Color Options (``buildtest report --color``)` on line 23. Please move this up.

Reply:
moved